lib/jekyll/configuration.rb

```diff
@@ -99,7 +99,7 @@ class << self
       # problems and backwards-compatibility.
       def from(user_config)
         Utils.deep_merge_hashes(DEFAULTS, Configuration[user_config].stringify_keys)
-          .add_default_collections
+          .add_default_collections.sanitize_url!.sanitize_baseurl!
```

Comment:
Is it more readable if each of these is on a separate line?

Reply:
Yes definitely. But, Rubocop complains that the class has exceeded the number of lines allowed..